api/tools/src/main/java/org/pmiops/workbench/tools/cdrConfig/UpdateCdrConfig.java

```diff
@@ -0,0 +1,331 @@
+package org.pmiops.workbench.tools.cdrConfig;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.Maps;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.sql.Timestamp;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import org.pmiops.workbench.db.dao.AccessTierDao;
+import org.pmiops.workbench.db.dao.CdrVersionDao;
+import org.pmiops.workbench.db.dao.WorkspaceDao;
+import org.pmiops.workbench.db.model.DbAccessTier;
+import org.pmiops.workbench.db.model.DbCdrVersion;
+import org.pmiops.workbench.model.ArchivalStatus;
+import org.pmiops.workbench.tools.CommandLineToolConfig;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * See api/project.rb update-cdr-config. Reads Access Tiers and CDR versions from a JSON file and
+ * updates the database to match.
+ */
+@Configuration
+public class UpdateCdrConfig {
```

Comment:
This tool started its life as `UpdateCdrVersions` but I have heavily modified it here.